third_party/abseil-cpp/absl/log/internal/log_sink_set.cc

```diff
@@ -58,8 +58,14 @@ bool& ThreadIsLoggingStatus() {
 #ifdef ABSL_HAVE_THREAD_LOCAL
   ABSL_CONST_INIT thread_local bool thread_is_logging = false;
   return thread_is_logging;
+#else
+#if defined(STARBOARD)
+  // TODO: b/328310825 - pthread_key_t doesn't have a constant initializer in
+  // Starboard.
+  static pthread_key_t thread_is_logging_key;
```

Comment:
Are we leaving it uninitialized? I think we can try DCHECK to guard it. Or we can try to initialize it through a function that is called to initialize another static variable after that one or play with the `once` APIs.

Reply:
Is this something that needs to be done in this PR or is it something you could look at separately as part of b/328310825?